packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -21,13 +143,95 @@ export default {
 
 .c-mediaElement {
     display: flex;
+    width: 100%;
+}
+.c-mediaElement--stack {
+    flex-direction: column;
+}
+.c-mediaElement--stack.c-mediaElement--reverse {
+    flex-direction: column-reverse;
+}
+
+.c-mediaElement-content {
+    display: flex;
+    flex-direction: column;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    flex: 1 1 0;
+
+    &--center {
+        align-items: center;
+        text-align: center;
+    }
+    &--left {
+        align-items: flex-start;
+    }
+    &--right {
+        align-items: flex-end;
+    }
+}
+
+.c-mediaElement-title.c-mediaElement-fontSize {
+    &--sm {
+        @include font-size(heading-s);
+    }
+    &--md {
+        @include font-size(heading-m);
+    }
+    &--lg {
+        @include font-size(heading-l);
+        margin-bottom: spacing(x0.5);
+    }
+    &--xl {
+        @include font-size(heading-xl);
+        margin-bottom: spacing(x0.5);
+    }
+    &--xxl {
+        @include font-size(heading-xxl);
+        margin-bottom: spacing(x0.5);
+    }
+}
```

Comment:
We usually don't use shorthand Sass selectors, just because it's harder to do a `find` in text ediors when searching for a classname. So I'd update this to be:

```suggestion
.c-mediaElement-title {
    &.c-mediaElement-fontSize--sm {
        @include font-size(heading-s);
    }
}
```

Reply:
See above commentt on sizing, as I think may make it a little bit cleaner with an outer modifer class.

Reply:
font sizing method has now changed, see below for comment, is based on your other suggestion.